### ISSUE TITLE
Check:imports command wrong reports when we use doc block

### ISSUE THREAD

**Issue by alikhosravidev** [imanghafoori1/php_token_analyzer#20]
PHP: v8.1
Laravel: v9
microscope: v1.0.318
Os: Pop Os

Autoload psr4:
"autoload": { "files": [ "app/helpers.php" ], "psr-4": { "App\\": "app/", "Database\\Factories\\": "database/factories/", "Database\\Seeders\\": "database/seeders/" } },

```php
<?php

namespace App\Core\BI\View\Widgets;

use App\Core\User\Entities\User;
use App\Core\User\Facades\UserService;
use Illuminate\View\Component;

class UserInsight extends Component
{
    public function render()
    {
        /* @var User $user */
        $user = UserService::user();

        return view(
            'modules.dashboard.widgets.user-insight',
                compact('user')
        );
    }
}

```

Reported message:
1 Extra Import:
App\Core\User\Entities\User
at app/Core/BI/View/Widgets/UserInsight.php:5

**Comment by imanghafoori1** [comment]
You have to start your doc-block with two stars, like this:
`/** @var Usr $u */`
and not:
`/* @var Usr $u */`
